### PR TITLE
Review project

### PR DESCRIPTION
**Pull request by Laylo309**
added microverse folder with:
index.html and style.css file;
added README.md file

### CODE REVIEW

**Comment by Laylooo** [comment]
> added microverse folder with:
> index.html and style.css file;
> added README.md file

- [ ] I think it is better to write a meaningful title for your request. Because sometimes after a long time you would get lost what did you do 😊😊😊
- [ ] Why don't you give descriptive commits? Other group members may don't understand what you have done and it may cause some problems. To avoid this, I suggest you make your commits summarizes all of the tasks you have done.👌
- [ ] _Check the comments under the review._
## Optional Suggestions

**Review by Laylo309 — COMMENTED** [review]
- [ ] I think it is better to write a meaningful title for your request. Because sometimes after a long time you would get lost what did you do 😊😊😊
- [ ] Why don't you give descriptive commits? Other group members may don't understand what you have done and it may cause some problems. To avoid this, I suggest you make your commits summarizes all of the tasks you have done.👌
- [ ] _Check the comments under the review._
## Optional Suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  margin-top: 6px;
+  margin-left: 25px;
+}
+
+.header nav {
+  margin-top: 10px;
+  margin-bottom: 10px;
+  width: 100%;
+  display: flex;
+}
```
Comment (Laylo309) [inline css/style.css:51]: I recommend you leave the line after finishing your code. Sometimes it causes some linter errors and you would probably be stuck in with it.😄

File: index.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    </form>
+
+  </div>
+
+
+
+
+  <!--
+
+    Hello Camper!
```
Comment (Laylo309) [inline index.html:55]: - [ ] you can remove this part of the code. Because those lines are unnecessary and you can get confused :blush:

**Review by smcommits — APPROVED** [review]
### Status: Approved 👍🏽 

Hi @Laylooo, 

## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

## Optional

- [ ] **You can improve the overall structure of the review comment. It will help with better understanding.**

You can improve the overall structures by including heading, break, etc. Please look into Github Markdown to understand more about it. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**